Assets/MixedRealityToolkit.Tests/PlayModeTests/SlateTests.cs

```diff
@@ -0,0 +1,177 @@
+
+            Assert.AreEqual(0.1, totalPanDelta.y, 0.05, "pan delta is not correct");
+
+            yield return h.Hide();
+        }
+
+        /// <summary>
+        /// Test hand ray scroll instantiated from prefab
+        /// </summary>
+        [UnityTest]
+        public IEnumerator Prefab_RayScroll()
+        {
+            InstantiateFromPrefab(Vector3.forward);
+            Vector2 totalPanDelta = Vector2.zero;
+            panZoom.PanUpdated.AddListener((hpd) => totalPanDelta += hpd.PanDelta);
+
+            TestHand h = new TestHand(Handedness.Right);
+            Vector3 screenPoint = CameraCache.Main.ViewportToScreenPoint(new Vector3(0.5f, 0.25f, 0.5f));
+            yield return h.Show(CameraCache.Main.ScreenToWorldPoint(screenPoint));
+
+            yield return h.SetGesture(ArticulatedHandPose.GestureId.Pinch);
+            yield return h.Move(new Vector3(0, -0.05f, 0), 10);
+            yield return h.SetGesture(ArticulatedHandPose.GestureId.Open);
+
+            Assert.AreEqual(0.1, totalPanDelta.y, 0.05, "pan delta is not correct");
+
+            yield return h.Hide();
+        }
+
+        /// <summary>
+        /// Test hand ray scroll instantiated from prefab
+        /// </summary>
+        [UnityTest]
+        public IEnumerator Prefab_TouchZoom()
+        {
+            InstantiateFromPrefab(Vector3.forward);
+
+            TestHand h = new TestHand(Handedness.Right);
+            yield return h.Show(Vector3.zero);
+
+            yield return h.MoveTo(panZoom.transform.position, 10);
+
+            TestHand h2 = new TestHand(Handedness.Left);
+            yield return h2.Show(Vector3.zero);
+            yield return h2.MoveTo(panZoom.transform.position + Vector3.right * -0.01f, 10);
+
+            yield return h.Move(new Vector3(0.01f, 0f, 0f));
+            yield return h2.Move(new Vector3(-0.01f, 0f, 0f));
+            Assert.AreEqual(0.4, panZoom.CurrentScale, 0.1, "slate did not zoom in using two finger touch");
+
+            yield return h.Hide();
+        }
+
+        /// <summary>
+        /// Test hand ray scroll instantiated from prefab
+        /// </summary>
+        [UnityTest]
+        public IEnumerator Prefab_GGVScroll()
+        {
+            InstantiateFromPrefab(Vector3.forward);
+            yield return RunGGVScrollTest(0.2f);
+        }
+
+        /// <summary>
+        /// Test hand ray scroll instantiated from prefab
+        /// </summary>
+        [UnityTest]
+        public IEnumerator Instantiate_GGVScroll()
+        {
+            InstantiateFromCode(Vector3.forward);
+            yield return RunGGVScrollTest(0.08f);
+        }
+
+        private IEnumerator RunGGVScrollTest(float expectedScroll)
+        {
+            PlayModeTestUtilities.SetHandSimulationMode(HandSimulationMode.Gestures);
+
+            Vector2 totalPanDelta = Vector2.zero;
+            panZoom.PanUpdated.AddListener((hpd) => totalPanDelta += hpd.PanDelta);
```

Comment:
add comment explaining you needed to do this because currentpanposition actually gives you the current pan DELTA (I should update the naming of that property in HIPZ).